config/fcos/v1_6_exp/translate.go

```diff
@@ -367,3 +375,68 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func (c Config) handleSelinux(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	rendered := types.Config{}
+	ts := translate.NewTranslationSet("yaml", "json")
+	var r report.Report
+	yamlPath := path.New("yaml", "selinux", "module")
+
+	hasValidModule := false
+
+	for _, module := range c.Selinux.Module {
+		if module.Name != "" && module.Content == "" || module.Content != "" && module.Name == "" || module.Name == "" && module.Content == "" {
+			r.AddOnWarn(path.New("yaml", "selinux", "module"), common.ErrFieldInvalid)
+		} else {
+			hasValidModule = true
+			break
+		}
+
+		if hasValidModule {
+			rendered.Storage.Filesystems = append(rendered.Storage.Filesystems,
+				types.Filesystem{
+					Device: "/dev/disk/by-label/boot",
+					Format: util.StrToPtr("ext4"),
+					Path:   util.StrToPtr("/boot"),
+				})
```

Comment:
So we only want this to happen once; but now it will happen for every valid module ? Im pretty sure we only want one filesystem.

Reply:
You are correct, I removed that part of creating the file system from inside the for loop. I think makes more sense now.